pyface/resource/resource_manager.py

```diff
@@ -238,3 +256,80 @@ def _get_default_resource_path(self, object):
                 break
 
         return resource_path
+
+
+def _get_package_data(module, rel_path):
+    """ Return package data in bytes for the given module and resource path.
+
+    Parameters
+    ----------
+    module : ModuleType
+        A module from which package data will be discovered.
+        If the module name does not conform to the package requirement, then
+        its "__file__" attribute is used for locating the directory to search
+        for resource files.
+    rel_path : str
+        "/"-separated path for loading data file.
+
+    Returns
+    -------
+    data : bytes
+        Loaded data in bytes.
+
+    Raises
+    ------
+    OSError
+        If the path referenced does not resolve to an existing file or the
+        file cannot be read.
+    """
+
+    if (module.__spec__ is None
+            or module.__spec__.submodule_search_locations is None):
+        module_dir_path = os.path.dirname(module.__file__)
+        path = os.path.join(module_dir_path, *rel_path.split("/"))
+        with open(path, "rb") as fp:
+            return fp.read()
+
+    return (
+        files(module.__name__).joinpath(rel_path).read_bytes()
```

Comment:
I believe this can be `files(module).joinpath(rel_path).read_bytes()` (an optimization to avoid having to locate the module using the name again).